android/cpp/mlperf_driver.cc

```diff
@@ -102,7 +102,7 @@ void MlperfDriver::RunMLPerfTest(const std::string& mode, int min_query_count,
   } else {
     // Run MLPerf in SingleStream mode by default.
     mlperf_settings.scenario = ::mlperf::TestScenario::SingleStream;
-    mlperf_settings.single_stream_expected_latency_ns = 1000000;
+    mlperf_settings.single_stream_expected_latency_ns = 800000;
```

Comment:
@anhappdev please make 900,000 one for Samsung backend (unified app for Samsung to test).

Reply:
@AhmedTElthakeb @fatihcakirs 
This is the unified APK built with `mlperf_settings.single_stream_expected_latency_ns = 900000`
[2022-05-10_PR287_mlperf_app.apk](https://drive.google.com/file/d/1pHvJu0AIoKxZNtY2qDmqSwdZ6v0Tdq-S/view?usp=sharing)